conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /mdg-mock              app.Routes
+->         /mdg-stub              app.Routes
```

Comment:
Do we know what the real endpoint is going to be? The stub endpoint should be identical.

Reply:
Nope :/